BUILDING.md

```diff
@@ -5,6 +5,10 @@
 * Protocol Buffers library from Google
 * QtCore from Qt 5
 
+On Debian/Ubuntu, these packages can be installed with:
+
+    apt install qt5-qmake qt5-default qt-sdk libprotobuf-dev protobuf-compiler libprotoc-dev
```

Comment:
The full qt-sdk package should not be required, only the Qt Core module is used.